scripts/apps/search/components/fields/associatedItems.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {IArticle} from 'superdesk-interfaces/Article';
+
+const TYPES_MAPPING = {
```

Comment:
Use clearer naming like `TypeToIconMap`. It's not clear now what does it map to.